src/main/java/cn/nukkit/network/protocol/StartGamePacket.java

```diff
@@ -92,8 +93,8 @@ public byte pid() {
     public boolean commandsEnabled;
     public boolean isTexturePacksRequired = false;
     public GameRules gameRules;
-    public boolean bonusChest = false;
-    public boolean hasStartWithMapEnabled = false;
+    public boolean bonusChestEnabled = false;
+    public boolean startingWithMap = false;
```

Comment:
Please, don't rename these fields to preserve binary compatibility with Cloudburst Nukkit

Reply:
ok